db/log_reader.cc

```diff
@@ -245,7 +258,7 @@ void Reader::ReportDrop(size_t bytes, const Status& reason) {
 }
 
 unsigned int Reader::ReadPhysicalRecord(Slice* result,
```

Comment:
Can we keep this as a flag ? This is just a primitive file reader, it might be better to keep it that way.